adapters/goldbach/params_test.go

```diff
@@ -0,0 +1,62 @@
+package goldbach
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+)
+
+// This file actually intends to test static/bidder-params/goldbach.json
+
+// TestValidParams makes sure that the goldbach schema accepts all imp.ext fields which we intend to support.
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
```

Comment:
```suggestion
	require.Error(t, err, "Failed to fetch the json-schemas. %v", err)
```

Do you mind to change other `IF... t.Errorf`  to `require.Error`?